rules/manager.go

```diff
@@ -377,6 +395,7 @@ func (g *Group) Eval(ctx context.Context, ts time.Time) {
 				sp.Finish()
 				evalDuration.Observe(time.Since(t).Seconds())
 				rule.SetEvaluationDuration(time.Since(t))
+				rule.SetEvaluationTimestamp(time.Now())
```

Comment:
This line should be `rule.SetEvaluationTimestamp(t)`.

Reply:
Same comment as above.